src/rendering/webgl/programs/node.circle.ts

```diff
@@ -72,6 +58,6 @@ export default class NodeCircleProgram extends NodeProgram<typeof UNIFORMS[numbe
     gl.uniform1f(u_correctionRatio, params.correctionRatio);
     gl.uniformMatrix3fv(u_matrix, false, params.matrix);
 
-    gl.drawArrays(gl.TRIANGLES, 0, this.verticesCount);
+    this.drawWebGL(gl.TRIANGLES);
```

Comment:
Maybe there is something to be done wrt naming to avoid the confusion between draw, drawWebGL and such?

Reply:
After merging branch [v3-1370-1371](https://github.com/jacomyal/sigma.js/tree/v3-1370-1371) the naming will be better.